Akash/html/assign 4/assig2.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+              table, th, td { border: 10px solid grey;
+                              border-collapse: collapse;
+                              text-align: center;
+                              border-bottom: 2px solid black; 
+                              border-right: 10px solid grey; 
+                              border: 2px solid black;
+                              border-right: 10px solid grey;
+                            } 
```

Comment:
this bracket should be below tables tmeansclosing bracket should be on the same line